Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -319,7 +319,14 @@ void ABenchmarkGymGameMode::SpawnNPC(const FVector& SpawnLocation, const FBlackb
 		return;
 	}
 
-	FVector FixedSpawnLocation = SpawnLocation;
+	const float RandomSpawnOffset = 600.0f;
+	FVector RandomOffset = FMath::VRand()*RandomSpawnOffset;
```

Comment:
What's all this about?

Reply:
Random offset for spawning NPCs - they all spawn on top of each other and on top of the players also. I removed this in a previous PR but it seems to be quite beneficial (the players and npcs flew off into the sky)

Reply:
Feature?